gradle.properties

```diff
@@ -108,12 +108,15 @@ artifactRedirecting.androidx.compose.foundation.version=1.7.0-beta05
 artifactRedirecting.androidx.compose.material.version=1.7.0-beta05
 # Look for `COMPOSE_MATERIAL3_COMMON` in libraryversions.toml
 artifactRedirecting.androidx.compose.material3.common.version=1.0.0-alpha01
+# Look for `COMPOSE_MATERIAL3_ADAPTIVE` in libraryversions.toml
+artifactRedirecting.androidx.compose.material3.adaptive.version=1.0.0-alpha01
```

Comment:
We didn't take into account that material3 adaptive has own release cycle.

Window-core has a separate release cycle as well.

Because of that we need:
- find a commit for the latest version
- partially merge subfolders of this commit to jb-main.

Please, create tasks on me:
- merge the latest material3.adaptive
- merge the latest androidx.window.core
- enable publication of material3.adaptive/androidx.window.core

Until we merge them, these specified versions can be unaligned with the actual state of the source, and we can't publish.

Let's remove also changes in `mpp/build.gradle.kts`, then we can merge the PR in the current state.

Reply:
>  then we can merge the PR in the current state.

Actually it might be better to merge it after, but I am not sure when I able to merge these folders.